src/routes/(app)/admin[[hash=admin_hash]]/age-retriction/+page.server.ts

```diff
@@ -0,0 +1,40 @@
+import { collections } from '$lib/server/database';
+import { runtimeConfig } from '$lib/server/runtime-config.js';
+import { set } from 'lodash-es';
+import type { JsonObject } from 'type-fest';
+import { z } from 'zod';
+
+export async function load() {}
+
+export const actions = {
+	update: async function ({ request }) {
+		const formData = await request.formData();
+		const json: JsonObject = {};
+
+		for (const [key, value] of formData) {
+			if (value) {
+				set(json, key, value);
+			}
+		}
+		const result = z
+			.object({
+				ageRestriction: z.object({ enabled: z.boolean({ coerce: true }), legalReason: z.string() })
```

Comment:
hm can we do just two values instead of a nested object?

Then we can do regular form data, no need for JSON tricks